### PR TITLE
🌐 [i18n-KO] Translated `add_new_pipeline.md` to Korean

### PR DESCRIPTION
**Pull request by heuristicwave**
# What does this PR do?

Translated the `add_new_pipeline.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

May you please review this PR? @nuatmochoi, @bolizabeth, @hyunhp, @mjk0618, @jungnerd

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

May you please review this PR? @sgugger, @ArthurZucker, @eunseojo

### CODE REVIEW

File: docs/source/ko/add_new_pipeline.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    if "maybe_arg" in kwargs:
+        preprocess_kwargs["maybe_arg"] = kwargs["maybe_arg"]
+
+    postprocess_kwargs = {}
+    if "top_k" in kwargs:
+        postprocess_kwargs["top_k"] = kwargs["top_k"]
+    return preprocess_kwargs, {}, postprocess_kwargs
+```
+
+입/출력을 가능한한 간단하고 완전히 JSON 직렬화 가능한 형식으로 유지하려고 노력하십시오.
```
Comment (nuatmochoi) [inline docs/source/ko/add_new_pipeline.md:109]:
```suggestion
입/출력을 가능한 한 간단하고 완전히 JSON 직렬화 가능한 형식으로 유지하려고 노력하십시오.
```
맞춤법 수정하였습니다.

**Review by nuatmochoi — CHANGES_REQUESTED** [review]

File: docs/source/ko/_toctree.yml [hunk 2]
```diff
@@ -128,7 +128,7 @@
     - local: perf_infer_gpu_one
       title: 하나의 GPU를 활용한 추론
     - local: perf_infer_gpu_many
-      title: 여러 GPU에서 추론
+      title: 다중 GPU에서 추론
```
Comment (wonhyeongseo) [inline docs/source/ko/_toctree.yml:131]: 고쳐주셔서 정말 감사합니다..! 👍

File: docs/source/ko/add_new_pipeline.md [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# 어떻게 사용자 정의 파이프라인을 생성하나요? [[how-to-create-a-custom-pipeline]]
+
+이 가이드에서는 사용자 정의 파이프라인을 어떻게 생성하고 [허브](hf.co/models)에 공유하거나 🤗 Transformers 라이브러리에 추가하는 방법을 살펴보겠습니다.
+
+먼저, 파이프라인이 수용할 수 있는 원시 입력을 결정해야 합니다.
```
Comment (wonhyeongseo) [inline docs/source/ko/add_new_pipeline.md:20]:
```suggestion
먼저 파이프라인이 수용할 수 있는 원시 입력을 결정해야 합니다.
```

File: docs/source/ko/add_new_pipeline.md [hunk 4]
```diff
@@ -13,21 +13,21 @@ rendered properly in your Markdown viewer.
...
+이 가이드에서는 사용자 정의 파이프라인을 어떻게 생성하고 [허브](hf.co/models)에 공유하거나 🤗 Transformers 라이브러리에 추가하는 방법을 살펴보겠습니다.
 
-First and foremost, you need to decide the raw entries the pipeline will be able to take. It can be strings, raw bytes,
-dictionaries or whatever seems to be the most likely desired input. Try to keep these inputs as pure Python as possible
-as it makes compatibility easier (even through other languages via JSON). Those will be the `inputs` of the
-pipeline (`preprocess`).
+먼저, 파이프라인이 수용할 수 있는 원시 입력을 결정해야 합니다.
+문자열, 원시 바이트, 딕셔너리 또는 가장 원하는 입력일 가능성이 높은 것이면 무엇이든 가능합니다.
+이 입력을 가능한 한 순수한 Python 형식으로 유지해야 호환성이 쉬워집니다(JSON을 통해 다른 언어와도 호환 가능).
+이러한 것들이 파이프라인(전처리)의 `inputs`이 될 것입니다.
```
Comment (wonhyeongseo) [inline docs/source/ko/add_new_pipeline.md:23]:
```suggestion
이 입력을 가능한 한 순수한 Python 형식으로 유지해야 (JSON을 통해 다른 언어와도) 호환성이 좋아집니다.
이것이 전처리(`preprocess`) 파이프라인의 입력(`inputs`)이 될 것입니다.
```

File: docs/source/ko/add_new_pipeline.md [hunk 5]
```diff
@@ -13,21 +13,21 @@ rendered properly in your Markdown viewer.
...
 
-Then define the `outputs`. Same policy as the `inputs`. The simpler, the better. Those will be the outputs of
-`postprocess` method.
+그런 다음 `outputs`를 정의합니다.
+`inputs`와 같은 정책을 따릅니다.
+간단할수록 좋습니다.
+이러한 것들이 `postprocess` 메소드의 출력이 될 것입니다.
```
Comment (wonhyeongseo) [inline docs/source/ko/add_new_pipeline.md:28]:
```suggestion
그런 다음 `outputs`를 정의하세요.
`inputs`와 같은 정책을 따르고, 간단할수록 좋습니다.
이것이 후처리(`postprocess`) 메소드의 출력이 될 것입니다.
```

File: docs/source/ko/add_new_pipeline.md [hunk 6]
```diff
@@ -130,22 +126,22 @@ PIPELINE_REGISTRY.register_pipeline(
...
     pipeline_class=MyPipeline,
     pt_model=AutoModelForSequenceClassification,
     default={"pt": ("user/awesome_model", "abcdef")},
-    type="text",  # current support type: text, audio, image, multimodal
+    type="text",  # 현재 지원 유형: text, audio, image, multimodal
 )
 ```
 
-## Share your pipeline on the Hub
+## 허브에 파이프라인 공유하기 [[share-your-pipeline-on-the-hub]]
```
Comment (wonhyeongseo) [inline docs/source/ko/add_new_pipeline.md:141]:
```suggestion
## Hub에 파이프라인 공유하기 [[share-your-pipeline-on-the-hub]]
```

File: docs/source/ko/add_new_pipeline.md [hunk 7]
```diff
@@ -130,22 +126,22 @@ PIPELINE_REGISTRY.register_pipeline(
...
 
-To share your custom pipeline on the Hub, you just have to save the custom code of your `Pipeline` subclass in a
-python file. For instance, let's say we want to use a custom pipeline for sentence pair classification like this:
+허브에 사용자 정의 파이프라인을 공유하려면 `Pipeline` 하위 클래스의 사용자 정의 코드를 Python 파일에 저장하기만 하면 됩니다.
```
Comment (wonhyeongseo) [inline docs/source/ko/add_new_pipeline.md:143]:
```suggestion
Hub에 사용자 정의 파이프라인을 공유하려면 `Pipeline` 하위 클래스의 사용자 정의 코드를 Python 파일에 저장하기만 하면 됩니다.
```

**Review by wonhyeongseo — CHANGES_REQUESTED** [review]
안녕하세요 @heuristicwave 님!

🔥 열정적으로 참여해주셔서 정말 감사드립니다. 전반적으로 고민하신 흔적이 보여서 너무 좋았어요! 아직 모두 리뷰를 못했지만 내일 OpenUP에서 마치겠습니다.

리뷰는 일방적인 수용이 아닌 서로간의 대화를 전제로 이루어집니다. 때문에 "어라, 이건 내가 맞아!"라고 주장해주시면 오히려 더 재밌는 리뷰 시간이 될 것입니다. 지금까지 최고의 리뷰를 꾸준히 올려주시는 @nuatmochoi 님 🥇 , 담당이신 @jungnerd 님도 모두 리뷰 자유롭게 올려주세요. 감사합니다. 💯 😆

**Comment by wonhyeongseo** [comment]
May you please review this PR @stevhliu ? cc: @sgugger , @ArthurZucker, @eunseojo
Thank you so much for your help!

**Review by stevhliu — APPROVED** [review]
Awesome, thank you for your contribution!

**Pull request merged**